docs/appendix/glossary/model-namespace-triplet.md

```diff
@@ -2,13 +2,24 @@
 title: Model Namespace Triplet
 id: model-namespace-triplet
 full_link:
-short_description: namespace:family:name
+short_description: namespace:repo-name:name or rdk:builtin:name
 aka:
 ---
 
-{{< glossary_tooltip term_id="model" text="Models" >}} are uniquely namespaced as colon-delimited-triplets in the form of `namespace:family:name`.
+{{< glossary_tooltip term_id="model" text="Models" >}} are uniquely namespaced as colon-delimited-triplets in the form of `namespace:repo-name:name` for modular resources and `rdk:builtin:name` for built-in models.
 
+We recommend to use the name of the git repository of your modular resource as the middle segment.
+The `repo-name` should describe the common functionality provided across the model or models of that module.
+
+For example:
+
+- The `rand:yahboom:arm` model uses the repository name [yahboom](https://github.com/viam-labs/yahboom).
+  It implements the `rdk:component:arm` API to support the yahboom dofbot arm.
```

Comment:
```suggestion
  It implements the `rdk:component:arm` API to support the Yahboom DOFBOT arm.
```

Reply:
Actually see below; I think rather than implementing these suggestions it'd be better to eliminate these examples and just link to the key concepts doc to avoid duplication.